src/renderer/components/layout/top-bar/top-bar.tsx

```diff
@@ -100,28 +97,24 @@ const NonInjectedTopBar = observer(({ items, children, isWindows, isLinux, ...re
         <Icon
           data-testid="home-button"
           material="home"
-          className="ml-4"
           onClick={goHome}
           disabled={isActiveRoute(catalogRoute)}
         />
         <Icon
           data-testid="history-back"
           material="arrow_back"
-          className="ml-5"
           onClick={goBack}
           disabled={!prevEnabled.get()}
         />
         <Icon
           data-testid="history-forward"
           material="arrow_forward"
-          className="ml-5"
           onClick={goForward}
           disabled={!nextEnabled.get()}
         />
       </div>
-      <div className={styles.controls}>
+      <div className={styles.items}>
         {renderRegisteredItems(items.get())}
```

Comment:
Note, it should now be relatively easy to add the ability to add items to either side via the extension API. That, however, is left for future work.